databricks/koala/structures.py

```diff
@@ -336,6 +360,14 @@ def _index_columns(self):
         return [anchor_wrap(self, self._spark_getitem(field))
                 for field in self._metadata.index_fields]
 
+    def _reduce_for_stat_function(self, sfun):
+        df = self._spark_select([sfun(self[col]).alias(col) for col in self.columns])
+        pdf = df.toPandas()
+        assert len(pdf) == 1, (df, pdf)
+        row = pdf.iloc[0]
+        row.name = None
+        return row  # Return first row as a Series
```

Comment:
@thunterdb Is it okay to return pandas Series here, or should we return `Column`?

Reply:
Since this is a row, it makes sense to return here either a Spark row or a pandas Series. Unless people have do complex operations with the Row, I like the idea of just returning a pandas series: it is compact and it should work both as a spark row substitute and as the expected pandas output.